docs/docs/rest-api/public/api/v2/schema/Group.json

```diff
@@ -23,6 +23,13 @@
         "$ref": "./AppDefinition.json"
       }
     },
+    "pods" : {
+      "type": "array",
+      "description": "The list of PodDefinitions in this group. See PodDefinition.json for the schema.",
+      "items": {
+        "$ref": "./PodDefinition.json"
```

Comment:
we'll probably migrate group.json to RAML format and here we'll reference the pod-status RAML type instead